### DISCUSSION TITLE
What are your opinions about the layout of the character information screen compared to other UIs?

### DISCUSSION THREAD

**Discussion by Flopblopper** [oobabooga/text-generation-webui#2660] [General]
I often switch between web UIs for LLMs. This one is my favorite, but I do find the character information screen to be kind of cluttered.
All of the character information besides the greeting is contained within a single text field, and I find that it makes editing a character more of a hassle than branching them out into separate fields. I can see advantages and disadvantages to doing it this way, and I was just curious about what you guys prefer. I've thought about making a feature request asking to consider the use of separate fields, but I imagine that it would be a huge change since it's been this way for so long, and it would be pointless to ask if I'm alone in preferring separate fields.

Maybe I'm overthinking this and most people just use a dedicated character creator or something.

**Comment by ossirytk** [comment]
I was thinking of this myself. The parser seems to ignore the description filed in the character file and folds the example_message into the context box. I was thinking of making a branch to fix this for my own use.  I wasn't sure if anyone cared and I thought the changes might add many new fields in the UI making it cluttered. I didn't want to piss of other people so I was going to make my own playground for this.
